src/ServerFactory.php

```diff
@@ -79,6 +79,10 @@ public function getCache()
             $cache = $this->config['cache'];
         }
 
+        if ($cache === false) {
+            return null;
```

Comment:
Functionally, you're correct in that `return;` is equivalent to `return null;`, but I like to be explicit in circumstances such as this. It implies that the caller of this method will be specifically checking for or using the null return.